bool marker_detected
float32[] prob
bbox[] bboxes
